packages/shadycss/src/style-transformer.js

```diff
@@ -315,6 +316,12 @@ class StyleTransformer {
           NTH, (m, type, inner) => `:${type}(${inner.replace(/\s/g, '')})`);
       selector = this._twiddleNthPlus(selector);
     }
+    if (!disableShadowParts && PART.test(selector)) {
+      // Hacky transform "::part(foo bar)" to "::part(foo,bar)" so that
+      // SIMPLE_SELECTOR_SEP isn't confused by the spaces.
+      // TODO(aomarks) Can we make SIMPLE_SELECTOR_SEP smarter instead?
```

Comment:
It would be nice to work this out, but acknowledge that it may be non-trivial.

Reply:
Acknowledged. Leaving for now with TODO.